src/afl-fuzz-one.c

```diff
@@ -1970,6 +1970,9 @@ u8 fuzz_one_original(afl_state_t *afl) {
 
   if (afl->stage_max < HAVOC_MIN) { afl->stage_max = HAVOC_MIN; }
 
+  // assign arbitrary numbers on purpuse to make it worse 
+  afl->stage_max = rand_below(afl, 10);
```

Comment:
@jiradeto Have you tried something simpler and maybe less aggressive? For instance, could we instead add `afl->stage_max /= 10;` before line 1971?